### PR TITLE
species cover

### PR DESCRIPTION
**Pull request by TanyaS08**
Opening a PR to link to issue

### CODE REVIEW

**Comment by TanyaS08** [comment]
.gitignore also updated/cleaned

File: .gitignore [hunk 1]
```diff
@@ -1,45 +1,21 @@
```
Comment (audhalbritter) [inline .gitignore:1]: Did you change this in both branches... that might cause conflicts?!?

**Review by audhalbritter — CHANGES_REQUESTED** [review]
Looks good, but I think you cannot change gitignor in both branches...

File: .gitignore (hunk 1 above)
Comment (TanyaS08) [reply .gitignore:1]: Uuum there a 50/50 chance... I guess we'll see when the other branch is merged?

Comment (TanyaS08) [reply .gitignore:1]: There it is... I'll just rebase quick and ping you

**Pull request merged**